cmd/snap/cmd_run_test.go

```diff
@@ -98,7 +98,7 @@ func (s *SnapSuite) TestSnapRunAppIntegration(c *check.C) {
 	// mock installed snap
 	dirs.SetRootDir(c.MkDir())
 	defer func() { dirs.SetRootDir("/") }()
-	defer mockSnapConfine()()
+	defer mockSnapConfine(dirs.DistroLibExecDir)()
```

Comment:
Aha, silly me. I even found this in subsequent reading but then forgot to delete the comment. Sorry for the noise.